syntax = "proto3";
package com.panelsw.ca;
option optimize_for = LITE_RUNTIME;

message UnzipDetails{

    bytes zipFile = 1;
    bytes targetFolder = 2;
}
